File: CSharp/Tests/BrokenBridgeTest.cs
using Xunit;

namespace CSharp.Tests
{
    public class BrokenBridgeTest
    {
        [Theory]
        [InlineData("####", true)]
        [InlineData("## ####", false)]
        [InlineData("#", true)]
        [InlineData("# #", false)]
        public void IsSafeBridge_StringValue_ReturnBoolean(string str, bool expected)
        {
            var actual = BrokenBridge.IsSafeBridge(str);

            Assert.Equal(expected, actual);
        }
    }
}

File: js/tests/brokenBridge.test.js
const isSafeBridge = require("../brokenBridge");

const data = [
  ["####", true],
  ["## ####", false],
  ["#", true],
  ["# #", false]
];

describe("isSafeBridge", () => {
  describe("Return boolean if bridge is safe to cross", () => {
    test.each(data)("isSafeBridge(%p) = %s", (str, expected) => {
      var actual = isSafeBridge(str);

      expect(actual).toEqual(expected);
    });
  });
});
